### DISCUSSION TITLE
Unable to install Rowy

### DISCUSSION THREAD

**Discussion by khaitanvaibhav** [AntlerVC/firetable#1039] [Support / Q&A] [answered]
I have been getting issues with getting my Rowy account on.
As a result, I deleted my workspace. I guess it did not delete at all.

Now that the workspace is deleted, I cannot add the Firebase to a new Rowy workspace. I get the error that 

**This project has already been connected to Rowy. Please select a different one.**

I am at my wits end to what should be done.

**Comment by harinij** [comment, marked as answer]
Hi @khaitanvaibhav did you use the delete workspace functionality on Rowy or did you delete it directly on your Firebase? Could you email us at support@rowy.io with the email id of the account you used - so we can check on it? Thanks.

**Reply by BeeBombshell** [reply]
Hey guys, closing this discussion since it has been unattended for a long time. @khaitanvaibhav if you're still facing this issue, feel free to create a new one.
